src/main/java/mate/Main.java

```diff
@@ -0,0 +1,18 @@
+package mate;
```

Comment:
I am just curious why did you change "mate.academy" to just "mate"

Reply:
just for simple understand